### PR TITLE
English translation reviewed

### PR DESCRIPTION
**Pull request by rasulksirov**

### CODE REVIEW

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 1]
```diff
@@ -1,194 +1,186 @@
-# First Hackathon on BEM
+# Our First BEM Hackathon
 
 Hi everyone,
 
-First hackathon on BEM took place from Nov 14 to 16, 2014, in Moscow. It took us long to go from an idea to the actual 
-realisation, and we would like to share our experience, results and plans with you.
+The first BEM hackathon took place in Moscow, on 14 - 16 November, 2014. It took us long to go from an idea to the actual realization, and we would now like to share our experience, achievements and plans for the future with you.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: achievements and plans –> achievements, and plans

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 2]
```diff
@@ -1,194 +1,186 @@
...
 
-A year and a half ago we started to move away from doing conferences and started to get together on small 
-[BEMups]( https://events.yandex.ru/events/bemup/). We called our meetups so, and run already 6 of them in Moscow, 
-Saint Petersburg, Minsk. Meetups gave us a lot: simple way of communication, meeting real people that use our technology, 
-an option to get to know everyone in person, discuss real life problems in any other case, do not limit our time for 
-questioning because of time limitations or a number of people.
+A year and a half ago we started to move away from conferences towards the smaller scale format of [BEMups]( https://events.yandex.ru/events/bemup/) - BEM meetups, six of which have since been held in Moscow, Saint Petersburg and Minsk. Such meetups gave us a lot: informal communication, opportunities to meet real people that use our technology, the chance to get to know everyone in person and discuss real-life problems related to specific products - and we no longer had to deal with certain restrictions, such as limited time for Q&A sessions, inevitable in conferences due to the sheer size of the audience.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: ”- BEM meetups“  –> should be an m dash –
”specific products - and we no longer “ –> should be an m dash –
Petersburg and Minsk –> Petersburg, and Minsk

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 3]
```diff
@@ -1,194 +1,186 @@
...
+We ended up with 8 teams out of 12 projects presented, which proceeded to work on their projects. They had just over 24 hours to code. Discussing tasks, deadlines, planning, interim results on an improvised kanban — we had it all. In every corner of our co-working space you would run into groups of people talking, drawing charts and diagrams on whiteboards, writing code, and occasionally popping out to take a nap in the hammock room :)
 
 ![](https://img-fotki.yandex.ru/get/15593/44214498.bc/0_9bbe3_ccfc51f8_XL.jpg)
 
-Sunday after the lunch teams presented their projects and results of 2-days hackathon. The goal was to get maximum number of tasks to production level. Many teams finished their job on 75-80% done level, and we scheduled the publication of the results on [bem.info/blog](https://bem.info/blog).
+On the Sunday afternoon the teams presented their projects and results of the two-day hackathon. The goal was to get the maximum number of tasks up to production level. Many teams finished their job at 75-80% "done" level - we have scheduled the publication of the results on [bem.info/blog](https://bem.info/blog).
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: "done" –> ”done“
level - we have –> m dash –

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 4]
```diff
@@ -1,194 +1,186 @@
...
+We announced our hackathon 2.5 months in advance - a significant period of time to keep registration open. However it was unavoidable. The selection process was labourious, applicants had to show their own BEM products or pass a test, we had to review their code, make sure nothing was missing and start to communicate with prospective participants. In other words, make all the necessary arrangements, cover travel expenses, sort out accommodation. Mentors of the projects had to think of tasks and goals, plan teamwork, try to foresee possible questions and think of the answers to them. We were not bored at all.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: m dash –

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 5]
```diff
@@ -1,194 +1,186 @@
...
 
-As the result, we had **34 calls for participation**. 23 invitations were sent after the review, 15 participants came to take part in the event. Five of them were not from Moscow but came from **Saint Petersburg, Orenburg, Voronezh, Ekaterinburg**. Our colleagues at Yandex were interested as well and came to code with us. 13 Yandex developers were in teams. We had **14 mentors** (some of the teams had two mentors at the same time), 4 of them were not from Yandex.
+We received **34 applications** in total. 23 invitations were sent out after the review, 15 participants attended the event. 5 of them were not from Moscow but came from **Saint Petersburg, Orenburg, Voronezh, Ekaterinburg**. Our colleagues at Yandex showed interest as well and came to code with us. The teams included 13 Yandex developers. We had **14 mentors** (some larger teams had two mentors), 4 of them were not from Yandex.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: **Saint Petersburg**, **Orenburg**, **Voronezh**, **Ekaterinburg**

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 6]
```diff
@@ -1,194 +1,186 @@
...
 - Documentation
-- A service that converts code into images with syntax highlighted written on BEM
-- MaximeOnline: content management system on BEM
-- BEM application for Smart-TV
+- A service that converts code into images with BEM syntax highlighted
+- MaximeOnline: content management system
+- BEM application for SmartTV
 
-How it was, what developers wanted to code and what they have got at the end will be written by mentors and published in our blog in the nearest future!
+How everything went, what goals developers pursued and what they achieved in the end - all of that will be chronicled by our mentors and published in our blog in the nearest future!
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: end - all of that –> m dash –

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 7]
```diff
@@ -1,194 +1,186 @@
...
+We had both mentors and participants taking part in the survey at a ratio of 40 to 60. **100 percent of respondents felt satisfied** with their teams, mentors, participants. Almost 100 percent were also satisfied with the level of preparation in the area of technology. Admittedly, we did follow a rather rigourous selection process at the registration stage, making sure that participants had the required level of BEM and JS knowledge given that the hackathon was about coding, not learning JS.
 
-We were worried however almost all teams passed planning period fast enough. They also planned tasks together as teams. At the end 80% of developers though of their work during hackathon as «realised everything we planned to code».
+Despite our worries, it didn't take long for most of the teams to do task planning and allocation. Task planning was done together, as a group effort in most of the teams. According to our survey, 80 percent of the developers believe they were able to "achieve all of the planned goals".
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: "achieve all of the planned goals" –> ” achieve all of the planned goals“

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 8]
```diff
@@ -1,194 +1,186 @@
...
 
-**Hackathon as a format** was found very comfortable and optimal. However there were developers that liked to start earlier or to sleep in the venue or to work more than two days or even not to leave it at all. We will listen to you definitely in the future!
+**The format of a hackathon** was generally found to be a very comfortable and optimal one. Although, some developers wish they'd been able to start a bit later, sleep in the venue, work for longer than two days or not have to leave at all. All of these suggestions will be taken into account in the future!
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: format of a hackathon –> format of the hackathon

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 9]
```diff
@@ -1,194 +1,186 @@
...
-- Vote for the ideas if there are too many, exclude unpopular to form teams of 5 developers for more productive realization of tasks
-- Look for more community-driven projects
+- Work on your skills beforehand so as not to slow down the team's progress
+- Configure the environment beforehand so as not to slow down the team's progress
+- Think the architecture through beforehand to save time
+- Estimate the complexity of the tasks, team effort, timelines; plan and break tasks down into subtasks if necessary to improve efficiency
+- Prepare projects with varying levels of difficulty — for beginners, to create really cool and useful stuff
+- Create projects that make use of the full BEM stack
+- If there is an excess of ideas, have a vote on all the projects proposed to narrow the list down so that we can have about 5 devs per team, which should lead to better results
+- Aim for more community-driven projects
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: кажется, по правилам английского нужна точка или в конце каждого пункта, или только в последнем.

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 10]
```diff
@@ -1,194 +1,186 @@
...
 
-Some of answers are be worth quoting:
+Let us also quote some of your comments:
 
-> «The main idea of any Hackathon is to have this atmosphere of creativeness and healthy competition. Therefore would be great to have more original projects from the community».
+> «The main idea of any hackathon is to have this atmosphere of creativity and healthy competition. So it would be great to have more original projects from the community».
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: здесь и ниже нужны английские кавычки ”...“

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 11]
```diff
@@ -1,194 +1,186 @@
...
+One respondent complains about a project the essence of which was unclear from the presentation, pointing out that the mentor had "spent two days doing nothing and the end result showed nothing." He advises us to avoid such kind of projects in the future. That's a good point. We will watch out for that kind of attitude in the future. This time we allowed ourselves to rely on the participants' self-discipline, their own enthusiasm for comminity building, and the fact that the rules of the event applied to all attendees, including the rule that states that "if no one joins your team and you are not likely to deliver the project in two days on your own, you should team up with another project to help others deliver on time". After a few reminders, queries and recommendations to join other teams, we decided not to push too hard and just see what happens. Next time we shall be stricter.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: quotes ”spent two days...“ and ”if no one joins your team...“

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 12]
```diff
@@ -1,194 +1,186 @@
...
 
-Another wrote us that "there was sad enough situation: only two teams outside of Yandex that were trying to code something and got 0 on voting". He suggested that we had to support them somehow. In his point of view, ideas were ok, but there were no technical skills and marketing. He also think that blocks and existing project development are strange ideas fot a hackathon".
+Another participant points to the "rather disappointing fact that the only two non-Yandex-based teams that were making some effort trying to code something got zero points in the voting." He suggests we show them some support. Their ideas were OK, in his view, the team was somewhat lacking in technical and marketing skills. He also finds blocks and the development of existing projects to be a strange idea for a hackathon.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: quotes ”...“

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 13]
```diff
@@ -1,194 +1,186 @@
...
 
-And here we fully disagree. Our hackathon was a project for the community where we could code together and opensource projects helping other BEM-developers. Half of 8 projects was exactly about it — Blocks (that needed by everyone), Building (modules for external building tools to build BEM-projects comfortably), Documentation (for the technologies that everyone use) and Forum (through which we communicate). These teams got the most of participants from the community. They were followed by 4 external projects — Dashboard, Pictures, Maxim Online and SmartTV.
+Now, we couldn't disagree more with that view and here's why. Our hackathon was planned as a project for the community, where we can code together to create and opensource something that will be of use to many developers. Half of the eight projects presented at the event were exactly about that - Blocks (which everyone needs), Build (modules for third-party build tools that facilitate the building of BEM projects), Documentation (for the technologies used by everyone) and Forum (where people discuss BEM). These teams were staffed mainly by people from the community.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: that - Blocks –> m dash –

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 14]
```diff
@@ -1,194 +1,186 @@
...
 
-According to the [voting results](http://instagram.com/p/vdvuzDjYhP/?modal=true) Dashboard got the first place with 11 votes,  Pictures got the 4th place together with Documentation team. Maxim Online and SmartTV got 0. It is unpleasant indeed, but not biased. The results that these teams presented were palpably lower than the others. We wish these teams next time to concentrate on better planning and better presenting, or to join other teams. It has sense — you will know more, will train your skills, will do better for the community, will bring efficiency to the project.
+Next, there were four external projects - Dashboard, Pictures, Maxim and SmartTV. The [results of the vote](http://instagram.com/p/vdvuzDjYhP/?modal=true) show that Dashboard won first place with 11 votes, Pictures shared fourth place with the Documentation team. Maxim and SmartTV both got zero points. That is indeed disappointing but it is hardly biased, since, as we all know, all the participants voted after all the projects were presented to them. The latter two teams showed poor results compared to the others.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: projects - Dashboard –> m dash –

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 15]
```diff
@@ -1,194 +1,186 @@
...
 
-![](https://img-fotki.yandex.ru/get/16122/44214498.bb/0_9bb8d_139fee96_XL.jpg)
+All we can do is suggest these teams improve their task planning skills, prepare their presentations better, work harder on their projects during the hackathon or maybe join other teams - which can also make sense - you end up gaining some knowledge and experience, doing you bit for the community and helping a lot of developers.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: teams - which can also make sense - you –> m dash

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 16]
```diff
@@ -1,194 +1,186 @@
...
 
+![](https://img-fotki.yandex.ru/get/16122/44214498.bb/0_9bb8d_139fee96_XL.jpg)
 
-We had no prizes for hackathon. While planning we thought that **our idea of making BEM world better** will drive us. Do you think the same way? Answers say that you do. Many said that they could code without prizes because they are happy to commit or that promotional prizes are enough. However, we will try to encourage you more in the future.
+Our hackathon did not involve prizes as such. While planning it, we thought we should be **driven by the idea to make the BEM world better**. Does everyone share that view? Your feedback certainly tells us so. Many share the view of "no prizes are needed, I'm happy to contribute to the common cause" or that "having promotional prizes is enough". However, we'll think of ways to encourage you more in the future.
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: quotes ”...“

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 17]
```diff
@@ -1,194 +1,186 @@
...
+- Knowledge and experience sharing
+- The ability to write blocks correctly
+- Acquiring skills on the go during the hackathon and from experienced developers
+- Getting to know their colleagues
+- Finding answers to their BEM questions and contributing to BEM development
+- Finding necessary information about further development and plans
+- Building ties with the BEM community and establishing contacts
+- Taking part in real BEM projects
+- Watching how others manage at the hackathon
+- Learning how to manage the process
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: точка или в конце всего списка или в конце каждого пункта. Одинаково во всем тексте.

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 18]
```diff
@@ -1,194 +1,186 @@
...
 
-Somebody wrote us that "he learnt to find answers about i-bem.js, tried to write tests, got useful comments on codestyle, learnt to notice things that thought unnecessary before. Because of hackathon he understood finally what needed to be improved and defined his professional direction for the next one to two years. Got a lot of benefits. Finished his part even though it was a biut small for the experienced developers".
+Someone wrote to us saying that he'd "learnt to find answers to questions about i-bem.js, tried his hand at writing tests, received some useful feedback on his coding style, learnt to notice things he'd treated as unimportant before". The hackathon helped him "identify the gaps in his knowledge as well as his professional development path for the next couple of years. So pluses all around really. Besides, I was able to complete my share of work, as straighforward as it was from an experienced developer perspective".
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: ”...“

File: blog/first-bem-hack/first-bem-hack.en.md [hunk 19]
```diff
@@ -1,194 +1,186 @@
...
 
-And that is cool!
+Sounds cool.
 
 ![](https://img-fotki.yandex.ru/get/16142/44214498.bc/0_9bbf6_afb95b1_XL.jpg)
 
 ## Afterword
 
-As an afterword I would like to quote some more of your emotions. Because that is exactly what has made our weekend!
+By way of afterword I would like to quote some more of your emotions - in the end, that was what really made our weekend!
```
Comment (innabelaya) [inline blog/first-bem-hack/first-bem-hack.en.md]: m dash

**Comment by rasulksirov** [comment]
Thanks vm for the comments, all corrections applied (serial commas, em dashes, quotes etc)

**Comment by tadatuta** [comment]
merged, thanks!

**Pull request merged**